packages/astro/src/runtime/client/dev-toolbar/apps/settings.ts

```diff
@@ -128,6 +150,27 @@ export default {
 					a:hover {
 						color: #f4ecfd;
 					}
+
+					select {
+						appearance: none;
+						text-align-last: center;
+						display: inline-block;
+						font-family: inherit;
+						font-size: 14px;
+						padding: 4px 24px 4px 8px;
+						border: 1px solid rgba(145, 152, 173, 1);
+						border-radius: 4px;
+						color: rgba(191, 193, 201, 1);
+						background-color: transparent;
+						background-image:
+							linear-gradient(45deg, transparent 50%, rgba(191, 193, 201, 1) 50%),
+							linear-gradient(135deg, rgba(191, 193, 201, 1) 50%, transparent 50%);
+						background-position:
+							calc(100% - 12px) calc(1em - 2px),
+							calc(100% - 8px) calc(1em - 2px);
+						background-size: 4px 4px;
+						background-repeat: no-repeat;
+					}
```

Comment:
Could this be abstracted into a component people can use? Similar to how the toggle works.

Reply:
Thanks for pointing this out! I'll take a look at the toggle component 🙂

Reply:
I have [turned it into a custom element](https://github.com/withastro/astro/pull/10591/commits/7efe653ae7ef4602e70a34ea9bbbcf3d125e1fcc) and [allowed it to have colors](https://github.com/withastro/astro/pull/10591/commits/bba2d0d26e35169552badcb828223e49698ff4ec) 🎨